cmd/kpod/create.go

```diff
@@ -206,6 +207,12 @@ func createCmd(c *cli.Context) error {
 		return err
 	}
 
+	logrus.Debug("new container created ", ctr.ID())
+	if err := ctr.Init(); err != nil {
+		return err
```

Comment:
Will this be sufficient?  Should we do an errors.Wrapf with something like "unable to initialize the container"?

Reply:
Init() should already be giving good errors - we're expecting that to be one of the primary API endpoints